src/main/java/com/uber/profiling/Arguments.java

```diff
@@ -34,6 +34,7 @@
 
 public class Arguments {
     public final static String DEFAULT_APP_ID_REGEX = "application_[\\w_]+";
+    public final static String DEFAULT_NAME_ID_REGEX = "application_name_[\\w_]+";
```

Comment:
@acharneski - This was not needed as application name would not start with the name 'application_name_' Its the spark.app.name what ever user gives while submitting spark job.